Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -78,12 +100,24 @@ def setContext(self, font, feaFile, compiler=None):
         Returns the context namespace instance.
         """
         todo = set(self.features)
+        insertComments = None
         if self.mode == "skip":
+            insertComments = _collectInsertMarkers(feaFile, self.insertFeatureMarker)
+            # only keep insert marker comments for our features
```

Comment:
what if instead `_collectInsertMarkers` takes a set of tags to include in the collection, ignoring the ones not in `todo`? you would not need to prune them later

Reply:
Done.